### PR TITLE
Redact password argument value

### PR DESCRIPTION
**Pull request by marians**
Fixes https://github.com/giantswarm/giantswarm/issues/2108

This PR should eliminate passwords from being transmitted via the command line header with API calls.

### CODE REVIEW

File: commands/commands.go [hunk 1]
```diff
@@ -125,10 +125,30 @@ func randomRequestID() string {
...
+			args[index+1] = "REDACTED"
+		} else if len(args) > 1 && args[1] == "login" {
+			// this will explicitly only apply to the login command
+			if strings.HasPrefix(arg, "-p=") {
+				args[index] = "-p=REDACTED"
+			} else if arg == "-p" {
+				args[index+1] = "REDACTED"
+			}
+		}
+	}
```
Comment (xh3b4sd) [inline commands/commands.go]: There should actually be tests for such things.

File: commands/commands.go [hunk 2]
```diff
@@ -125,10 +125,30 @@ func randomRequestID() string {
...
+
+	fmt.Printf("Args: %#v\n", args)
```
Comment (xh3b4sd) [inline commands/commands.go]: I think this should be removed before merging.

**Comment by marians** [comment]
- Added a test
- Removed that print command
- Turned off CircleCI again, as it was never working anyway

**Comment by marians** [comment]
@xh3b4sd Do you approve?

**Pull request merged**